src/screens/CreatePost/elements/SheetCloseBtn.js

```diff
@@ -21,14 +23,14 @@ const SheetCloseBtn = ({backRef, goBack, continueToEdit}) => {
       <View style={styles.container}>
         <Text style={styles.header}>Are you sure?</Text>
         <Gap style={styles.gap(30)} />
-        <Button onPress={continueToEdit}>
+        <Button onPress={continueToEdit} style={{backgroundColor: dynamicColors.primary}}>
           <Text>Continue editing</Text>
         </Button>
         <Gap style={styles.gap(10)} />
         <Button
           onPress={goBack}
-          styles={{backgroundColor: colors.porcelain}}
-          textStyling={{color: colors.black}}>
+          style={{backgroundColor: COLORS.porcelain}}
+          textStyling={{color: COLORS.black}}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the text in the "Continue editing" button is not specified. This could lead to visibility issues depending on the `dynamicColors.primary` value. It's a good practice to always specify both background and text colors for readability.
```diff
- <Button onPress={continueToEdit} style={{backgroundColor: dynamicColors.primary}}>
+ <Button onPress={continueToEdit} style={{backgroundColor: dynamicColors.primary}} textStyling={{color: COLORS.black}}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->